am-lib/src/test/java/uk/gov/hmcts/reform/amlib/internal/utils/SecurityClassificationTest.java

```diff
@@ -42,4 +54,24 @@ void whenRoleSecurityClassificationIsHighEnoughResourceShouldBeVisible(boolean e
             Arguments.of(false, NONE, RESTRICTED)
         );
     }
+
+    private static Stream<FromValueArguments> fromValueArguments() {
+        return Stream.of(
+            new FromValueArguments(NONE, NONE),
+            new FromValueArguments(PUBLIC, NONE, PUBLIC),
+            new FromValueArguments(PRIVATE, NONE, PUBLIC, PRIVATE),
+            new FromValueArguments(RESTRICTED, NONE, PUBLIC, PRIVATE, RESTRICTED)
+        );
+    }
+
+    private static class FromValueArguments {
+        private final SecurityClassification roleSecurityClassification;
```

Comment:
I'd be keen to avoid using word `role` here, although this is just example usage so might be acceptable.